config/releases.exs

```diff
@@ -0,0 +1,31 @@
+import Config
+
+config :ex_aws,
+  dynamodb: [
+    # make sure port is first, otherwise it won't be converted to an integer
+    port: System.get_env("DYNAMO_PORT"),
```

Comment:
I think you can make this `String.to_integer(System.get_env("DYNAMO_PORT"))` now, since it's running as an Elixir script.